frontend/src/static/js/v_segment.js

```diff
@@ -3,24 +3,14 @@ window.vSegment = {
   template: window.$('v_segment').innerHTML,
   data() {
     return {
-      loaded: this.segment.authored || this.segment.lines.length < 5,
+      isOpen: this.segment.authored || this.segment.lines.length < 5,
+      hasCloser: !this.segment.authored && this.segment.lines.length > 4,
```

Comment:
`hasCloser` kind of seems like a misnomer. Can name it more appropriately?

Reply:
I can't really think of a better name since its a boolean which determines if the component has a `closer` which I got from how the html class is named. If I remember from my cs2103 booleans usually should start with 'is' or 'has', google says 'can' is usually used too, so would `canOpen` be a better name?

Reply:
Both OK to me.

Reply:
`canOpen` sounds better to me.

Reply:
@gerhean Can you fix the issue of this comment?